scripts/test_all

```diff
@@ -50,14 +50,7 @@ did_any_fail=false
 
 xcodebuid_test \
   -workspace "$ROOT_DIR/catalog/MDCCatalog.xcworkspace" \
-  -scheme "MaterialComponents-Unit-Tests"
-if [ $? -ne 0 ]; then
-  did_any_fail=true
-fi
-
-xcodebuid_test \
-  -workspace "$ROOT_DIR/catalog/MDCCatalog.xcworkspace" \
-  -scheme "MaterialComponentsAlpha-Unit-Tests"
+  -scheme "MDCCatalog"
```

Comment:
Was this an intentional change? It looks unrelated to the screenshot tests.

Reply:
This was intentional because the `MaterialComponentsAlpha-Unit-Tests` is generated by cocoapods but we need to set an Environment variable on that scheme. So instead we will run the tests that are in the MDCCatalog scheme. This will include the `MaterialComponents-Unit-Tests`, `MaterialComponentsAlpha-Unit-Tests`, and the new`MaterialComponentsSnapshotTests-Unit-Tests` schemes and simplify our script.